lib/shopify_app/sessions_controller.rb

```diff
@@ -12,8 +12,10 @@ def create
 
     def callback
       if response = request.env['omniauth.auth']
-        sess = ShopifyAPI::Session.new(params[:shop], response['credentials']['token'])
+        shop_name = params[:shop]
```

Comment:
Pretty sure there was a security issue taking the shop name from the params instead of from the omniauth hash. Ping @Shopify/security @dylanahsmith  (for reference: https://github.com/Shopify/shopify-theme-store/pull/494)